### PR TITLE
Hide index / deletion fields

### LINKED ISSUE #544: Hide index & deletion fields on formsets
Pretty sure that we used to hide these... not 100% sure when this... broke? Should get fixed, in any case.

### PR DESCRIPTION
**Pull request by melinath**
Resolved #544. Turns out the only organizer forms that displayed these fields were the custom form forms. So I cleaned them up a bit and made them match the item option formsets, resolving this issue along the way.

### CODE REVIEW

File: brambling/templates/brambling/event/organizer/_item_form.html [hunk 1]
```diff
@@ -75,7 +75,7 @@
 		</div>
 		<div class="panel-footer">
 			<div class='js-formset-add option-add'>
-			</div>{# /.list-group-item #}
```
Comment (harrislapiroff) [inline brambling/templates/brambling/event/organizer/_item_form.html]: wow.

**Comment by harrislapiroff** [comment]
ummm how do I delete a field tho

**Comment by melinath** [comment]
*facepalm* I will fix that.

**Comment by harrislapiroff** [comment]
:stuck_out_tongue_closed_eyes: :D

File: brambling/templates/brambling/forms/customformfield.html [hunk 2]
```diff
@@ -4,18 +4,46 @@
...
-	{% formrow form.choices %}
-	<div class='row js-hide'>
-		<div class='col-md-6'>{% formrow form.index %}</div>
-		<div class='col-md-6'>{% formrow form.DELETE %}</div>
+	<div class='row'>
+		<div class='col-sm-1 hidden-xs'>
+			<span class="js-sortable-handle btn btn-link btn-block a-secret pull-right"><i class="fa fa-fw fa-bars"></i></span>
+		</div>
+		<div class='margin-trailer-half-xs col-sm-4'>
+			{% formrow form.name using "brambling/forms/rows/no_label.html" with placeholder="Do you snore?" %}
```
Comment (harrislapiroff) [inline brambling/templates/brambling/forms/customformfield.html]: I don't love having this as the placeholder for every field but I might let it go till we have a superior form editing interface.

**Pull request merged**